### DISCUSSION TITLE
How can I contribute to CNCF with my teck stack in a DevOps related project

### DISCUSSION THREAD

**Discussion by VaibhavJak** [cncf/mentoring#661] [Q&A] [answered]
Hi all. I am a student of IIIT Lucknow. I am very new to open source but am familiar with the basics of networking, docker, Kubernetes, and Linux commands. Is there any way that I can contribute to CNCF and get hands-on experience in a real DevOps project? As a matter of fact, I am aiming for GSOC 2023. My email id: jakkulavaibhav@gmail.com

**Comment by nate-double-u** [comment, marked as answer]
Hi @VaibhavJak!

If you want to be a Cloud Native contributor, I suggest starting by checking out the [CNCF Contributors](https://contribute.cncf.io/) page.

If you're interested in mentorship I think that GSOC is a great program. You can also take a look around this repo and see what opportunities there are. Some projects also have issues marked "Good first issue" which you can take a look at. It's good to communicate and work with projects that you're interested in doing a mentorship with.

**Reply by VaibhavJak** [reply]
Hi @nate-double-u . I see .. Appreciate your quick response. Thanks:)

**Comment by abs007** [comment]
It seems like [Service Mesh Performance](https://contribute.cncf.io/contributors/projects/#service-mesh-performance) might be a good fit for you.
